app/ota_metadata.py

```diff
@@ -271,6 +273,23 @@ def __init__(self, info):
 
         del self._left
 
+    def __eq__(self, _other: Any) -> bool:
+        if isinstance(_other, DirectoryInf):
+            return _other.path == self.path
+        elif isinstance(_other, Path):
+            return _other == self.path
+        else:
+            return False
+
+    def __hash__(self) -> int:
+        return hash(self.path)
+
+    def mkdir2slot(self, dst_root: Path):
```

Comment:
In this file, the name `slot` is inappropriate. Name `slot` doesn't exist in this file context.

Reply:
I will unify the naming in the ota_metadata to make the function name more understandable.

Reply:
Thanks :pray: